tests/topotests/bgp_snmp_bgp4v2_notification/test_bgp_snmp_bgp4v2_notification.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2024 6WIND S.A.
+#
+
+
+"""
+Test BGP OPEN NOTIFY (Configuration mismatch) followed by snmpwalk.
+"""
+
+import os
+import sys
+import json
+from time import sleep
```

Comment:
Could you drop this import, it's not used.

Reply:
fixed